remote-build-coordinator/src/main/java/org/jboss/pnc/remotecoordinator/rexclient/MyHeaderPropagator.java

```diff
@@ -41,12 +41,6 @@ public MultivaluedMap<String, String> update(
             MultivaluedMap<String, String> clientOutgoingHeaders) {
         MultivaluedMap<String, String> outgoingHeaders = new MultivaluedHashMap<>();
 
-        // propagate User Token if present in original request
-        if (incomingHeaders.containsKey(HttpHeaders.AUTHORIZATION_STRING)) {
```

Comment:
In this place, there should be Authorization header from Orch's SA

Reply:
@michalovjan Thanks for the pointer! Done :)